core/container/src/main/java/org/wildfly/swarm/Swarm.java

```diff
@@ -265,11 +336,31 @@ public Swarm component(Class<?> cls) {
         return this;
     }
 
+    /**
+     * Add an outbound socket-binding to the container.
+     *
+     * <p>In the even the specified {@code socketBindingGroup} does no exist, the socket-binding
```

Comment:
even -> event, does no exist -> does not exist

This also appears in the `socketBinding` method.